src/main/java/com/nashtech/exception/ApiError.java

```diff
@@ -0,0 +1,25 @@
+package com.nashtech.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+public class ApiError {
```

Comment:
Add "message" variable in ApiError class and rename "errorCode" to "httpStatusCode".